packages/storage/src/reference.ts

```diff
@@ -118,46 +123,90 @@ export class Reference {
 }
 
 /**
- * Uploads a blob to this object's location.
+ * Uploads data to this object's location.
+ * The upload is not resumable.
  * @public
- * @param ref - Storage Reference where data should be uploaded.
+ * @param ref - StorageReference where data should be uploaded.
  * @param data - The data to upload.
- * @param metadata - Metadata for the newly uploaded string.
+ * @param metadata - Metadata for the newly uploaded data.
+ * @returns An UploadTaskNonResumableSnapshot
+ */
+export function uploadBytes(
+  ref: StorageReference,
+  data: Blob | Uint8Array | ArrayBuffer,
+  metadata?: Metadata
+): Promise<UploadTaskSnapshot> {
+  ref._throwIfRoot('uploadBytes');
+  return _nonResumableUpload(ref, data, metadata);
+}
+
+/**
+ * Uploads data to this object's location.
+ * The upload is resumable and exposes progress updates.
```

Comment:
```suggestion
 * The upload can be paused, resumed and exposes progress updates.
```

Suggesting this because the Web SDK doesn't actually allow you to resume an upload with an Upload URL (as far as I can tell). Instead, the user can only resume uploads with an UploadTask that they paused.

Android let's users actually resume an upload with an Upload URL: https://firebase.google.com/docs/reference/android/com/google/firebase/storage/StorageReference#putFile(android.net.Uri,%20com.google.firebase.storage.StorageMetadata,%20android.net.Uri)

Reply:
Changed.